src/docs/product/data-management-settings/filtering/index.mdx

```diff
@@ -16,14 +16,36 @@ These filters are exclusively applied at ingest time and not later in processing
 
 Inbound filters include:
 
-- Common browser extension errors
-- Events coming from localhost
-- Known legacy browsers errors
-- Known web crawlers
-- By their error message
-- From specific release versions of your code
-- From certain IP addresses
-- Transactions coming from healthcheck and ping requests
+- **Common browser extension errors**
+- **Events coming from localhost**
+- **Known legacy browsers errors**
+- **Known web crawlers**
+- **By their error message**
+- **From specific release versions of your code**
+- **From certain IP addresses**
+- **Transactions coming from healthcheck and ping requests**
+
+### Transactions coming from healthcheck
+
+In essence, the health check filter serves the purpose of excluding transactions that are generated as a part of health check procedures.
+
+By applying this filter, you effectively bypass transactions that, while crucial for app stability assessment, hold limited value for you beyond their designated function.
+
+We consider a transaction to be a health check if its name matches one of the following glob patterns:
+
+- `*healthcheck*`
+- `*healthy*`
+- `*live*`
+- `*ready*`
+- `*heartbeat*`
+- `*/health`
+- `*/healthz`
+- `*/ping`
+
+<hr />
+
+A node will be red if it is associated with errors.
+Click "View Full Trace" to display the [Trace View](/product/sentry-basics/tracing/trace-view).
```

Comment:
This feels like a sudden change of topic. It's unclear to me what nodes and traces you're talking about here.

Reply:
I noticed the same but this was already in the docs I don't have the context to change it... I would prefer to leave it there as it is